bpf/include/bpf/ctx/xdp.h

```diff
@@ -206,6 +206,45 @@ l4_csum_replace(struct xdp_md *ctx, __u64 off, __u32 from, __u32 to,
 	return ret;
 }
 
+static __always_inline __maybe_unused int
+ctx_adjust_room(struct xdp_md *ctx, const __s32 len_diff, const __u32 mode,
+		const __u64 flags)
+{
+	const __u32 move_len_v4 = 14 + 20;
```

Comment:
```suggestion
	const __u32 move_len_v4 = sizeof(struct ethhdr)+ sizeof(struct iphdr);
	...
```
?

Reply:
Given it's a generic header, I didn't want to pull in networking specific includes in here.